resources/gtfo.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Hmm good idea, this file is going to change a bunch. Specially since I think we're going to be using refresh tokens for the handshake, not access tokens